core/tests/build_sources/templates/base.html

```diff
@@ -78,8 +78,6 @@
 
     <script>
       var GLOBALS = {
-        ADDITIONAL_ANGULAR_MODULES: JSON.parse(
-          '{{additional_angular_modules|js_string}}'),
```

Comment:
Remove also the other globals in this file.

Reply:
Done